src/main.ts

```diff
@@ -1,14 +1,221 @@
+                if (accessToken) {
+                    operation.setContext({
+                        headers: new HttpHeaders().set(AppValues.HEADERS_AUTHORIZATION_KEY, `Bearer ${accessToken}`),
+                    });
+                }
+                return forward(operation);
+            });
+
+            const globalLoaderMiddleware: ApolloLink = new ApolloLink((operation: Operation, forward: NextLink) => {
+                const context = operation.getContext();
+                const skipLoading = Boolean(context.skipLoading);
+                const headers = context.headers as HttpHeaders;
+                const headersExist = headers && headers.keys().length;
+                if (skipLoading) {
+                    operation.setContext({
+                        headers: headersExist
+                            ? headers.append(AppValues.HEADERS_SKIP_LOADING_KEY, `${skipLoading}`)
+                            : new HttpHeaders().set(AppValues.HEADERS_SKIP_LOADING_KEY, `${skipLoading}`),
+                    });
+                }
+                return forward(operation);
+            });
+
+            return {
+                cache: apolloCache(),
+                link: ApolloLink.from([errorMiddleware, authorizationMiddleware, globalLoaderMiddleware, httpMainLink]),
+            };
+        },
+        deps: [HttpLink, AppConfigService, LocalStorageService, Injector],
+    },
+    {
+        provide: HIGHLIGHT_OPTIONS,
+        useValue: {
+            coreLibraryLoader: () => import("highlight.js/lib/core"),
+            languages: {
+                sql: () => import("highlight.js/lib/languages/sql"),
+                yaml: () => import("highlight.js/lib/languages/yaml"),
+            },
+        },
+    },
+    {
+        provide: APP_INITIALIZER,
+        useFactory: (loggedUserService: LoggedUserService) => {
+            return () => {
+                return loggedUserService.initializeCompletes();
+            };
+        },
+        deps: [LoggedUserService],
+        multi: true,
+    },
+    {
+        provide: APP_INITIALIZER,
+        useFactory: (loginService: LoginService) => {
+            return (): Promise<void> => {
+                return firstValueFrom(loginService.initialize()).catch((e) => logError(e));
+            };
+        },
+        deps: [LoginService],
+        multi: true,
+    },
+    {
+        provide: MAT_RIPPLE_GLOBAL_OPTIONS,
+        useValue: {
+            disabled: true,
+        },
+    },
+];
 
 if (environment.production) {
     enableProdMode();
 }
 
-platformBrowserDynamic()
-    .bootstrapModule(AppModule)
+bootstrapApplication(AppComponent, {
+    providers: [
+        importProvidersFrom(
+            CommonModule,
+            BrowserModule,
+            ClipboardModule,
+            MarkdownModule.forRoot({
+                loader: HttpClient,
+                sanitize: SecurityContext.NONE,
+            }),
+            ModalModule.forRoot(),
```

Comment:
This is also our module, eliminate this.

Reply:
Removed